queries.sql

```diff
@@ -51,3 +51,27 @@ SELECT neutered, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY neutered;
 /* Average number of escape attempts per animal type of those born between 1990 and 2000 */
 SELECT neutered, AVG(escape_attempts) FROM animals WHERE date_of_birth 
 BETWEEN 'Jan 1, 1990' AND 'Dec 31, 2000' GROUP BY neutered;
+
+-- Write queries (using JOIN) :
+-- What animals belong to Melody Pond?
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE full_name='Melody Pond';
+--List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name FROM animals JOIN species ON animals.species_id=species.id WHERE species_id=1;
+
+--List all owners and their animals, remember to include those that don't own any animal.
+SELECT animals.name,full_name FROM owners LEFT JOIN animals ON animals.owner_id=owners.id;
+
+--How many animals are there per species?
+SELECT species.name, COUNT(*) from animals JOIN species ON animals.species_id=species.id GROUP BY species.name;
+
+--List all Digimon owned by Jennifer Orwell.
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id=owners.id WHERE owners.full_name='Jennifer Orwell' AND species_id=2;
+
+
+--List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id=owners.id 
+WHERE owners.full_name='Dean Winchester' AND escape_attempts=0;
+
+--Who owns the most animals?
+SELECT owners.full_name, COUNT(animals.name) AS total FROM owners LEFT JOIN animals ON animals.owner_id=owners.id 
+GROUP BY owners.full_name ORDER BY total DESC LIMIT 1;
```

Comment:
- [ ] <b>REQUIRED:</b> You did not implement queries for assigning owners to animals as requested by the project. Kindly implement them. ex:

```sql
   UPDATE animals
   --You can use hardcoded owner_id
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith') 
   WHERE name = 'Agumon';
```